providers/harness/README.md

```diff
@@ -0,0 +1,65 @@
+# Unofficial Harness OpenFeature GO Provider
+
+ [Harness](https://developer.harness.io/docs/feature-flags) OpenFeature Provider can provide usage for Harness via OpenFeature GO SDK.
+
+# Installation
+
+To use the Harness provider, you'll need to install [Harness Go client](github.com/harness/ff-golang-server-sdk) and Harness provider. You can install the packages using the following command
+
+```shell
+go get github.com/harness/ff-golang-server-sdk
+go get github.com/open-feature/go-sdk-contrib/providers/harness
+```
+
+## Concepts
+* Provider Object evaluation gets Harness JSON evaluation.
+* Other provider types evaluation gets Harness matching type evaluation.
+
+## Usage
+Harness OpenFeature Provider is using Harness GO SDK.
+
+## Usage Example
+
+```go
+import (
+  harness "github.com/harness/ff-golang-server-sdk/client"
+  harnessProvider "github.com/open-feature/go-sdk-contrib/providers/harness/pkg"
+)
+
+providerConfig := harnessProvider.ProviderConfig{
+    Options: []harness.ConfigOption{
+        harness.WithWaitForInitialized(true),
+        harness.WithURL(URL),
+        harness.WithStreamEnabled(false),
+        harness.WithHTTPClient(http.DefaultClient),
+        harness.WithStoreEnabled(false),
+    },
+    SdkKey: ValidSDKKey,
+}
+
+provider, err := harnessProvider.NewProvider(providerConfig)
+if err != nil {
+    t.Fail()
+}
+err = provider.Init(of.EvaluationContext{})
+if err != nil {
+    t.Fail()
+}
+
+ctx := context.Background()
+
+target := map[string]interface{}{
```

Comment:
```suggestion
context := map[string]interface{}{
```

"target" makes sense, but I think `context` is probably best especially for documentation... unless "target" is a word Harness uses specifically, then I think this is fine.

Reply:
Nevermind! I can see that "target" is Harness' term.